tasks/old/ThemedFinals.tex

```diff
@@ -0,0 +1,90 @@
+
+Story-telling is an important factor, so it is recommended to spend the least amount of time using the microphone to explain the demonstration and let the demonstration speak for itself.
+
+
+\section{Evaluating Juries for Final Demonstrations}
+The \iterm{Finals} are evaluated by two juries, here described.
+
+\begin{enumerate}
+\item\textbf{League-internal jury:} The league-internal jury is formed by the Executive Committee. The evaluation of the league-internal jury is based on the following criteria:
+  \begin{compactenum}
+  \item Efficacy/elegance of the solution
+  \item Innovation/contribution to the league of the additional tasks solved
+  \item Difficulty of the overall demonstration
+  \end{compactenum}
+
+\item \textbf{League-external jury:} The league-external jury consists of people not being involved in the RoboCup@Home league, but having a related background (not necessarily robotics). They are appointed by the Executive Committee. The evaluation of the league-external jury is based on the following criteria:
+  \begin{compactenum}
+  \item Originality and presentation (story-telling is to be rewarded)
+  \item Relevance/usefulness to everyday life
+  \item Elegance/success of overall demonstration
+  \end{compactenum}
+\end{enumerate}
+
+\section{Scoring}
+The final score and ranking are determined by the jury evaluations and by the previous performance (in Stages I and II) of the team, in the following manner:
+
+\begin{enumerate}
+  \item The influence of the league-internal jury to the final ranking is \SI{25}{\percent}.
+  \item The influence of the league-external jury to the final ranking is \SI{25}{\percent}.
+  \item The influence of the total sum of points scored by the team in Stage I and II is \SI{50}{\percent}.
+\end{enumerate}
+
+These demonstrations are carried out in a serialized fashion, one League performing after another in one \Arena{}.
+
+
+\subsection{Task}
+The procedure for the demonstration and the timing of slots is as follows:
+\OpenDemonstrationTask{ten}{five}
+
+\OpenDemonstrationChanges
+
+%% %%%%%%%%%%%%%%%%%%%%%%%%
+\section{Final Ranking and Winner}
+
+There will be an award for 1st, 2nd and 3rd place of each league.
+
+The winner of the competition is the team that gets the highest ranking in the \iterm{Finals}.
+
+The second place will be the team that got the second-highest ranking in the \iterm{Finals}.
+
+The third place will be the team with the highest score that did not made it to the \iterm{Finals}.
+
+Additional certificates would be granted if:
+
+\begin{enumerate}
+  \item If the number of teams in the league is above 11, a certificate will be awarded to the 4th ranked team.
+  \item If the number of teams in the league is above 14, a certificate will be awarded to the 5th ranked team.
```

Comment:
See my comment above re award for 1, 2 & 3rd place.

Can you prove in a comment next to this the citation to the RoboCup Organization rules on awards vs certificates? I know in the past when SSPL had low number of teams registered we did not 3rd place awards.